src/Gateway/GlobalAuthenticationGateway.php

```diff
@@ -8,14 +10,14 @@
 
 class GlobalAuthenticationGateway extends ID3GlobalBaseGateway
 {
-    public function AuthenticateSP(string $profileID, int $profileVersion, ?string $customerReference, Identity $identity)
+    public function AuthenticateSP(string $pId, int $pVer, ?string $customerReference, Identity $identity): stdClass
```

Comment:
Keeping the older parameter names would help on the code readability. Actually, I do not see any benefit of shortening names here.

Reply:
Hey - yeah I agree, this was to get the line length under 120 characters, and just so I could avoid making a multi-line constructor as I just personally don't like the readability of those.

I'll adjust this (and the other places I've changed it) to be a multi-line constructor instead :)